config/index.js

```diff
@@ -19,7 +20,7 @@ const getConfig = () => {
 
   const customConfig = readConfig(environment)
 
-  const combinedConfig = Object.assign({}, defaultConfig, customConfig)
+  const combinedConfig = _.merge({}, defaultConfig, customConfig)
```

Comment:
Key line to fix the 404 issue seen in production!

Reply:
PR with config schema & validation coming soon. This will guard against using obviously invalid configuration.